### DISCUSSION TITLE
I cant make auto-save work since Firefox 119.0

### DISCUSSION THREAD

**Discussion by verysecrethero** [gildas-lormeau/SingleFile#1283] [General]
hello, auto-save has worked wonderfully but now since 119.0 it doesnt work
im also trying auto-save periodically which doesnt work

i hope you can make these work again, they are really really useful and i like this SingleFile a lot!
Thanks in advance!

**Comment by gildas-lormeau** [comment]
Hi, I cannot reproduce the issue. Did you change some settings in SingleFile?

**Comment by verysecrethero** [comment]
hello ok, the following 2 pages dont seem to work with auto-save but they did before 119.0:
https://www.youtube.com/feed/subscriptions
https://www.sciencedirect.com/search?date=2023&langs=en&sortBy=date&show=100&offset=0
also, periodically doesnt work for these pages
the option to manually save these pages works
the auto-save will make it appear in pending saves but no file appears(it isnt saved) on disk
ive recently tried some other webpages and some work good with auto-save
i did change some settings, however since you wrote it i hit the reset button and then reloaded the config, with no changes

hope that helps finding out what might be wrong thanks

**Reply by gildas-lormeau** [reply]
Thank you, I confirm I was able to reproduce and fix the issue with the first URL. I couldn't test the second one because the page is not public. The fix will be available in the next version which should hopefully being available for download in Firefox before the end of the week.

**Comment by verysecrethero** [comment]
thanks a lot!  good job with the fix! looking forward to it being available

**Reply by gildas-lormeau** [reply]
You're welcome :)

**Reply by gildas-lormeau** [reply]
@verysecrethero The update is available for download on the store of Mozilla, the timing was perfect ;)
